pkg/es/writer/esBulkHandler.go

```diff
@@ -318,3 +319,56 @@ func PostBulkErrorResponse(ctx *fasthttp.RequestCtx) {
 	responsebody["status"] = 400
 	utils.WriteJsonResponse(ctx, responsebody)
 }
+
+func ProcessDeleteIndex(ctx *fasthttp.RequestCtx, myid uint64) {
```

Comment:
It seems like this is accepting wildcard index names. Is that right? If so, can you write a comment saying that?